README.md

```diff
@@ -19,63 +19,4 @@
 
 # Apache APISIX Helm Charts
 
-## Dependencies
```

Comment:
Why removing these descriptions?

Reply:
This md move to `charts/apisix`.
see https://github.com/apache/apisix-helm-chart/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22

Reply:
Even so, link is necessary.

Reply:
It's also required for dashboard.

Reply:
see https://github.com/apache/apisix-helm-chart/pull/27/files#diff-bac1d31fa0f27c7026dcebb6994af021495e330e6fe54545d44b23937fb04e7cR20

Reply:
It's better to add links for these two charts in the README.

Reply:
> It's better to add links for these two charts in the README.

Appended.